tests/unit/test_project.py

```diff
@@ -50,13 +57,7 @@
 summary: "example for unit tests"
 description: "this is an example of a rockcraft.yaml for the purpose of testing rockcraft"
 license: Apache-2.0
-entrypoint: ["/bin/hello"]
-cmd: ["world"]
-env:
-    - NAME: "VALUE"
-package-repositories:
```

Comment:
wait why was the `package-repositories` entry removed?

Reply:
jeez...I've had a lot of conflicts when rebasing with main. Hopefully the last commit does it